drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -196,6 +203,7 @@ function DragNDropTemplates(url_name) {
                         'data-zone_id': zone.id,
                         'data-zone_align': zone.align,
                         'role': 'button',
+                        'aria-describedby': getItemPropertyFromSet(items_in_zone_descriptions, "displayName").join(", "),
```

Comment:
@dovinmu opinionated nit (feel free to argue): It might be a bit confusing - might make sense to add something like "items in the current zone: ...".